### PR TITLE
Some fixes from cloud version

### PR DESCRIPTION
**Pull request by parkererickson-tg**

### CODE REVIEW

File: .github/workflows/build-deploy-ci.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: CoPilot Build and Deploy
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "cloud-main" ]
```
Comment (RobRossmiller-TG) [inline .github/workflows/build-deploy-ci.yaml:5]: reset to main

**Review by RobRossmiller-TG — APPROVED** [review]
Looks good. Don't forget to reset workflow branch trigger

**Review by billshitg — APPROVED** [review]
Just to confirm: those issues were caught by cloud, but the fixes are general?

**Comment by parkererickson-tg** [comment]
Correct, they are general

**Pull request merged**